.github/workflows/test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           path: ~/.local/share/pnpm/
           key: pnpm-${{ hashFiles('pnpm-lock.yaml') }}
       - name: Install dependencies
-        run: pnpm install --frozen-lockfile --ignore-scripts
+        run: pnpm install --frozen-lockfile
```

Comment:
Too bad for security (`postinstall` scripts are the main source of npm dependencies attack).

Let’s call the script directly instead.

Reply:
Added `pnpm -r postinstall`

Reply:
Is there a way to call a specific script from client/server?

`cd client && pnpm postinstall` ?